fix: Print proper balance in wasp-cli wallet cmd

howjmay opened the pull request.

ivange94 commented on packages/metrics/chain.go:
```diff
@@ -83,6 +89,14 @@ func (c *chainMetricsObj) RecordBlockSize(blockIndex uint32, blockSize float64)
 	c.metrics.blockSizes.With(prometheus.Labels{"chain": c.chainID.String(), "block_index": fmt.Sprintf("%d", blockIndex)}).Set(blockSize)
 }
 
+func (c *chainMetricsObj) LastSeenStateIndex(stateIndex uint32) {
```
Never mind. I get it.

kape1395 replied:
`CurrentStateIndex` is the index for which next batch is being collected and agreed.
If a node gets stuck, it is possible that it will receive newer `LastSeenStateIndex` from the network while still working on the `CurrentStateIndex`.

howjmay committed:
fix: Print proper balance in wasp-cli wallet cmd

iotajsonrpc.Balance object doesn't have String() command, which
makes GlazedFormatter format object pointer's address into string,
but not the balance information
